### ISSUE TITLE
cmd/ebitenmobile: crash on iOS with go-inovation

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/ebiten#2577]
### Ebitengine Version

ad7d5a86f9f5e005740cbc1bfb16d482cc674ab1

### Operating System

- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] FreeBSD
- [ ] OpenBSD
- [ ] Android
- [X] iOS
- [ ] Nintendo Switch
- [ ] Xbox
- [ ] Web Browsers

### Go Version (`go version`)

go version go1.20 darwin/amd64

### What steps will reproduce the problem?

Build and run [`go-inovation`](https://github.com/hajimehoshi/go-inovation) for iOS

### What is the expected result?

No crash

### What happens instead?

Crash on launching

```
-[MTLDebugRenderCommandEncoder setScissorRect:]:3635: failed assertion `Set Scissor Rect Validation
(rect.y(0) + rect.height(192))(192) must be <= render pass height(64)
```

**Comment by hajimehoshi** [comment]
![image](https://user-images.githubusercontent.com/16950/220173861-6d930653-bca5-4da3-ae81-700aabc5e5d1.png)

There is another flaky crash but this might be a bug in go-inovation

**Comment by hajimehoshi** [comment]
```
961c34efb5da5b4517eae18178c9e9a7cd3b8c60 is the first bad commit
commit 961c34efb5da5b4517eae18178c9e9a7cd3b8c60
Author: Hajime Hoshi <hajimehoshi@gmail.com>
Date:   Sat Feb 18 18:34:06 2023 +0900

    internal/ui: use a small buffer image for anti-alias
    
    Closes #2399

 image_test.go        | 106 ++++++++++++++++++++++++-------
 internal/ui/image.go | 172 +++++++++++++++++++++++++++++++++++++++++++++------
 2 files changed, 235 insertions(+), 43 deletions(-)
```

**Comment by hajimehoshi** [comment]
Anti-aliasing is always used internally even though the game doesn't use it:

```
goroutine 1 [running, locked to thread]:
github.com/hajimehoshi/ebiten/v2/internal/ui.(*Image).DrawTriangles(0x1300e4230, {0x1300e40a0, 0x0, 0x0, 0x0}, {0x1301f5b80, 0x20, 0x20}, {0x1301ac100, 0x6, ...}, ...)
	/Users/hajimehoshi/ebiten/internal/ui/image.go:84 +0x210
github.com/hajimehoshi/ebiten/v2.(*Image).DrawTriangles(0x1302b2000, {0x1301f5b00, 0x4, 0x800000100255c?}, {0x1301ac0f0, 0x6, 0x4?}, 0x1302b2310, 0x0?)
	/Users/hajimehoshi/ebiten/image.go:500 +0x70c
github.com/hajimehoshi/ebiten/v2/vector.drawVerticesForUtil(0x130065a00?, {0x1301f5b00?, 0x4, 0x4}, {0x1301ac0f0, 0x6, 0x8}, {0x1027192a8?, 0x1301ac00c?})
	/Users/hajimehoshi/ebiten/vector/util.go:48 +0x118
github.com/hajimehoshi/ebiten/v2/vector.DrawFilledRect(0x1302bffff?, 0x40000000, 0x40000000, 0x41e00000, 0x41e00000, {0x1027192a8, 0x1301ac00c})
	/Users/hajimehoshi/ebiten/vector/util.go:72 +0x1a8
github.com/hajimehoshi/ebiten/v2/ebitenutil.DrawRect(...)
	/Users/hajimehoshi/ebiten/ebitenutil/shapes.go:49
github.com/hajimehoshi/go-inovation/ino/internal/draw.init.0()
	/Users/hajimehoshi/go-inovation/ino/internal/draw/draw.go:32 +0x90
```
